src/Model/ModelData.php

```diff
@@ -48,16 +45,18 @@ class ModelData
      * </code>
      */
     private static array $casting = [
-        'CSSClasses' => 'Varchar'
+        'CSSClasses' => 'Varchar',
+        'forTemplate' => 'HTMLText',
     ];
 
     /**
-     * The default object to cast scalar fields to if casting information is not specified, and casting to an object
+     * The default class to cast scalar fields to if casting information is not specified, and casting to an object
      * is required.
+     * This can be any injectable service name but must resolve to a DBField subclass.
+     *
+     * If null, casting will be determined based on the type of value (e.g. integers will be cast to DBInt)
      */
-    private static string $default_cast = 'Text';
-
-    private static array $casting_cache = [];
+    private static ?string $default_cast = null;
```

Comment:
`null` essentially means "use the appropriate `DBField` instance for scalars".

I thought about just removing this config altogether, but it could be useful in edge cases like a model where _all_ of the fields are meant to be HTML, for example.